libraries/ms-common/inc/flash.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+// Raw Flash memory API
+// Warning: Offers no protection against overwriting program code
```

Comment:
Theoretically you could add a variable into the linker script? I'd prefer not to do that because you could use this API as-is to implement a bootloader. :)